tests/excerptexport/test_downloads.py

```diff
@@ -24,7 +24,8 @@ def test_file_download(self, *args):
         excerpt = Excerpt.objects.create(name='Neverland', is_active=True, is_public=True, owner=user,
                                          bounding_geometry=bg)
         extraction_order = ExtractionOrder.objects.create(excerpt=excerpt, orderer=user)
-        output_file = OutputFile.objects.create(mime_type='test/plain', extraction_order=extraction_order,
+        export = extraction_order.exports.create(file_format='fgdb')
+        output_file = OutputFile.objects.create(mime_type='test/plain', export=export,
```

Comment:
I don't remember what we thought about `test/plain`. I think it's a typo copied two times.